options/options.c

```diff
@@ -32,6 +32,8 @@
 
 #ifdef _WIN32
 #include <windows.h>
+#if __MINGW64_VERSION_MAJOR >= 12
+#include <dwmapi.h>
```

Comment:
This should be in `w32_common`.

Reply:
Was a unclosed if anyway, due to my manual patching.. will be removed in next force push